app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_save :update_like_counter
+
+  def update_like_counter
+    post.update(LikesCounter: count)
+  end
```

Comment:
- [ ] The `count` variable you are using here is undefined, Please make sure you use valid code to update the counter. I would suggest you to using the below code.

```ruby
def update_like_counter
    post.increment!(:LikesCounter)
  end
```

- [ ] [Optional] I would suggest you make this method private So it can't be accessed outside of this class.